src/components/input.js

```diff
@@ -19,7 +19,10 @@ const styles = {
     position: 'fixed', top: 0, left: 0,
     maxHeight: 36 * 8 + 2, overflowY: 'auto',
     backgroundColor: 'white',
-    border: `1px solid ${colors.light()}`
+    border: `1px solid ${colors.light()}`,
+    margin: '8px 0',
+    borderRadius: '4px',
```

Comment:
nit: Typically I've seen in the codebase using REMs for margins and padding favored over px (when possible) - is there a reason for choosing to use pixels here?

Reply:
For border radii, px can be the way to go in order to match a mock, since small changes in number can have large visual impact.
```suggestion
    borderRadius: 4,
```

Reply:
+1 on Isaac's comment for border radii. On the margin I was matching the select styling. I changed this to (0.5) rem to be in-line with our margin settings elsewhere. Used suggestion as well.